ios/gutenberg/GutenbergViewController.swift

```diff
@@ -116,6 +116,15 @@ extension GutenbergViewController: GutenbergBridgeDelegate {
 }
 
 extension GutenbergViewController: GutenbergBridgeDataSource {
+    
+    func gutenbergLocale() -> String? {
+        return "en"
```

Comment:
We could send the device locale directly. Since is the example app is not a blocker, but it would nice to test translations just by changing the device language in the run settings for fast tests.
```suggestion
        return Locale.preferredLanguages.first ?? "en"
```